waku/v2/discv5/discover.go

```diff
@@ -212,8 +212,10 @@ func (d *DiscoveryV5) listen() error {
 
 	d.listener = listener
 
-	d.log.Info(fmt.Sprintf("Started Discovery V5 at %s:%d, advertising IP: %s:%d", d.udpAddr.IP, d.udpAddr.Port, d.localnode.Node().IP(), d.localnode.Node().TCP()))
-	d.log.Info("Discv5: discoverable ENR ", d.localnode.Node())
+	d.log.Info("started Discovery V5",
+		zap.Stringer("listening", d.udpAddr),
+		logging.TCPAddr("advertising", d.localnode.Node().IP(), d.localnode.Node().TCP()))
+	d.log.Info("Discovery V5: discoverable ENR ", logging.ENode("node", d.localnode.Node()))
```

Comment:
Calling the `ENode` attribute `node` is not good, since we're using it for the peer ID of the local node. But I'm unsure what to call the field. Would `enr` or `nodeRecord` or something be suitable? There are few of these sprinkled around but are easy to find (`logging.Enode(`).

Reply:
either `enr` or `nodeRecord` is fine!

Reply:
ok, `enr` it is then, I'll make the change shortly.

Reply:
Done